src/test/ServeFiles.java

```diff
@@ -1,33 +1,36 @@
 package test;
 
-import java.util.HashMap;
-import java.util.Random;
+import http.HttpContentType;
+import http.HttpRequest;
+import http.HttpResponse;
+import http.HttpStatusCode;
+import server.Server;
 
-import lib.HTTPCode;
 
-public class ServeFiles extends server.Server {
-    public static void main(String[] args) throws Exception {
-        ServeFiles SF = new ServeFiles();
-        SF.HTTPStart(); // -> For HTTP Servers (default port: 80)
-        /* For HTTPS Servers
-        SF.setPort(443);
-        SF.HTTPSStart("./etc/keystore.jks", "123456");
-        */ 
+import java.io.BufferedInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class ServeFiles {
+    private static HttpResponse handle(HttpRequest httpRequest) {
+        HttpResponse httpResponse = new HttpResponse();
+        String path = httpRequest.getPath().replaceFirst("/", "www/");
+        try (BufferedInputStream f = new BufferedInputStream(new FileInputStream(path))) {
+            httpResponse.setHttpStatusCode(HttpStatusCode.OK);
+            httpResponse.setHttpContentType(HttpContentType.TEXT_PLAIN);
+            byte[] fileContents = f.readAllBytes();
+            httpResponse.setBody(new String(fileContents, 0, fileContents.length, StandardCharsets.UTF_8));
```

Comment:
Why did you use String wrapper? What if it is a raw file (eg. mp4)?

Reply:
Please add JKS comment, in the SimpleLogin class as well

Reply:
> Why did you use String wrapper? What if it is a raw file (eg. mp4)?

I think this is resolved by the last commit.